### PR TITLE
Feature/update map slightly

### PR DESCRIPTION
**Pull request by JR-Mitchell**
Adds in some more roots to the map, and moves collectables

### CODE REVIEW

File: assets.wasmp [hunk 1]
```diff
@@ -59,7 +59,6 @@
 -- 002:0022000002c200000033300003220000002200000022c0000002200000022000
 -- 003:0022000000c220000333300000220000002200000022c0000002200000220000
 -- 004:00022000002c20000333300000222300002200000022c0000022000002220000
--- 005:00033000003d30000ffff00000333f00003300000033d0000033000003330000
```
Comment (JR-Mitchell) [inline assets.wasmp]: My bad, will revert this change

Comment (JR-Mitchell) [reply assets.wasmp]: Reverted in c00251c

**Review by fjebaker — APPROVED** [review]
Ja ja gut.
